test/regression

```diff
@@ -42,7 +42,6 @@ define_tool_script "tap" "src/tap/test/regression"
 define_tool_script "ppl" "src/ppl/test/regression"
 define_tool_script "pdn" "src/pdn/test/regression"
 define_tool_script "pad" "src/pad/test/regression"
-define_tool_script "mpl" "src/mpl/test/regression"
```

Comment:
are all the tests failing? we can also disable individual tests [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/mpl/test/regression_tests.tcl)

Reply:
I removed the test files specified in this issue https://github.com/The-OpenROAD-Project/OpenROAD/issues/2343